### ISSUE TITLE
Blocks have weird lines with shadows

### ISSUE THREAD

**Issue by xdomix723** [X0nk/Bliss-Shader#261]
Blocks will have a bunch of weird lines on them when there are shadows. I haven't seen this on a previous version of the shader but have just noticed it when updating to the latest build.

![image](https://github.com/user-attachments/assets/f8c72712-2515-4936-8211-8c96d22bedba)

NVIDIA
Windows
1.21
Vanilla Launcher
Iris
Bliss Main

The bug can be seen when at a certain distance away. It goes away when super close or when further away. 

The only shader setting I found to fix it was bringing up the resolution. What made it even worse was to turn up the shadow distance very high.

**Comment by jbritain** [comment]
This is called shadow acne and there is not much that can be done about it apart from messing with the shadow bias.

**Comment by joshua-igor** [comment]
It would be interesting to have the option to adjust Shadow Bias to one's liking in order to mitigate this.

**Comment by X0nk** [comment]
yeah, i changed the bias a bit a while ago trying to fine tune it, its really tricky getting it to adjust for different combinations of shadow resolution and shadow distance.

a setting to personally adjust it shouldnt be needed if i do it better.
